src/flow.ts

```diff
@@ -210,22 +210,48 @@ export class Flow {
    * Example:
    * ```
    * await Vex.Flow.fetchMusicFont('Petaluma');
-   * Vex.Flow.setMusicFont('Petaluma');
+   * Vex.Flow.setFont('Petaluma');
    * ... (do VexFlow stuff) ...
    * ```
    * See `demos/fonts/` for more examples.
    *
    * @returns an array of Font objects corresponding to the provided `fontNames`.
    */
-  static setMusicFont(...fontNames: string[]): void {
+  static setFont(...fontNames: string[]): void {
     // Convert the array of font names into an array of Font objects.
     MetricsDefaults.fontFamily = fontNames.join(',');
   }
 
-  static getMusicFont(): string[] {
+  static getFont(): string[] {
     return Metrics.get('fontFamily').split(',');
   }
 
+  /**
+   * Load the fonts that are used by your app.
+   * If `fontNames` is undefined, all fonts in Font.FILES will be loaded.
+   * This is useful for debugging, but not recommended for production because it will load lots of fonts.
+   *
+   * For example, on the `flow.html` test page, you could call:
+   *   `await Vex.Flow.loadFonts();`
+   *
+   * Alternatively, you may load web fonts with a stylesheet link (e.g., from Google Fonts),
+   * and a @font-face { font-family: ... } rule in your CSS.
+   *
+   * Customize `Font.HOST_URL` and `Font.FILES` to load different fonts for your app.
+   */
+  static async loadFonts(...fontNames: string[]): Promise<void> {
```

Comment:
`loadFonts(...fontNames)` is an alternative to the `@font-face` CSS rules. We can pass in a comma separated list of font names, and VexFlow will load the specified fonts from the CDN.